### ISSUE TITLE
Is it possible to put a react component in the cell of a table?

### ISSUE THREAD

**Issue by jkoren** [react-bootstrap-table/react-bootstrap-table2#1737]
I am experimenting with react-bootstrap-table2 for an open source application.

**Question**
Is it possible to put a react component in the cell of a table?  I have a table that looks like the attached screenshot, and am thinking about incorporating react-bootstrap-table2 to allow sorting of columns.

![image](https://user-images.githubusercontent.com/67333843/163509458-5b586e74-9098-4915-ae77-6d6eb0292e6a.png)

You can see the code [here](https://github.com/jkoren/advocacy-maps/blob/fbc83c213d49fbe4311bd28b45205cab7335b93b/components/ViewBills/ViewBills.js#L182) with the original table and the start of my react-bootstrap-table2 below it.

here is a [working demo without react-bootstrap-table-2](https://digital-testimony-dev.web.app/bills)
.

**Comment by Ymbere** [comment]
Yes, I think that what you are looking for is [Row formatter](https://react-bootstrap-table.github.io/react-bootstrap-table2/docs/column-props.html#columnformatter-function).

This is a minimum example using it

[![Edit column-sort-test](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/column-sort-test-iiiu4?fontsize=14&hidenavigation=1&theme=dark)
